flask_mapper/flask_mapper/templates/map.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+       integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ=="
+          crossorigin=""/>
+    <!-- Make sure you put this AFTER Leaflet's CSS -->
+     <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"
+                integrity="sha512-/Nsx9X4HebavoBvEBuyp3I7od5tA0UzAxs+j83KgC8PU0kgB4XiK4Lfe4y4cgBtaRJQEIFCW+oC506aPT2L1zw=="
+                   crossorigin=""></script>
+
+    </head>
+    <body>
+        <div id="mapid" style="height: 600px"></div>
+        <script>
```

Comment:
Inline js directly within the `body` is generally frowned upon. Scripts should be broken out into separate files and included in the `head`.

Reply:
This was actually intentional for two reasons.  I didn't want the overhead of another server call, especially because this would be iframed in rather than a discrete page.  The other reason was to keep the routing as simple as possible.  This js is the only part of the page that is generated dynamically, so I didn't want to have to add a second route for a js file.

Reply:
Okay, I understand the hesitation for a second route. In that case, I'd use the body `onload` event that references a function defined in a head / script tag.

Reply:
ok cool, got it